sdk/program/src/account_info.rs

```diff
@@ -394,6 +396,111 @@ impl<'a> AsRef<AccountInfo<'a>> for AccountInfo<'a> {
     }
 }
 
+#[derive(Clone)]
+pub struct VmSliceTemp<T> {
```

Comment:
If this is the main place where `VmSlice` will be used, then it seems fine to define it here with the other types.

Reply:
Yeah, I think that's where it will end up, and it can be pulled into the other code that uses it from here. Will clean that up at the end.